### PR TITLE
Wedge elements

### PR DESCRIPTION
**Pull request by vlukes**
@rc, please look at this PR. The volume integration seems to be working, the surface integration not yet. Problem is in FEField.create_mapping(), where faces of the same type are assumed. Do you have any idea how to get around this easily?

### CODE REVIEW

**Comment by rc** [comment]
Thanks! Ad facet integration: maybe define two regions, one for each face type? The basis function arrays (`bf`, `bfg`) need to be rectangular. A possible hack would be to expand triangular faces to quadrangles, and the extra node would have zero basis. Like:
```
0, 1, 2 -> 0, 1, 2, 2
N_0, N_1, N_2 -> N_0, N_1, N_2, 0_of_shape_N_1
```
??

**Comment by rc** [comment]
Do you plan to add an example using the wedge elements?

**Comment by vlukes** [comment]
> Do you plan to add an example using the wedge elements?

Yes, I plan to add an example comparing deformations of a hexa/tetra/wedge beam.

**Comment by vlukes** [comment]
Done. I hope :)

**Comment by rc** [comment]
Thanks!

Does setting the integral order in the example to more than 1 work for you? It errors for me.

**Comment by vlukes** [comment]
Yes, you are right. There was just one quadrature rule and a bug in FEMapping.create_mapping().

New quadrature rules are generated by [this code](https://people.math.sc.edu/Burkardt/m_src/wedge_felippa_rule/wedge_felippa_rule.html).

File: sfepy/discrete/fem/mappings.py [hunk 1]
```diff
@@ -257,6 +304,10 @@ def get_mapping(self, qp_coors, weights, bf=None, poly_space=None,
             The transformation matrix applied to the basis functions.
         is_face: bool
             Is it the boundary of a region?
+        fc_bf_map: tuple
+            The additional info to remap face derivatives of wedge elements:
+            - fc_bf_map[0]: id of face group (triangle or quid)
```
Comment (rc) [inline sfepy/discrete/fem/mappings.py:309]: quid -> quad

File: sfepy/discrete/fem/mappings.py [hunk 2]
```diff
@@ -257,6 +304,10 @@ def get_mapping(self, qp_coors, weights, bf=None, poly_space=None,
...
+            - fc_bf_map[1]: position of inplane deravatives (xy-axes)
```
Comment (rc) [inline sfepy/discrete/fem/mappings.py:310]: deravatives -> derivatives

File: sfepy/examples/linear_elasticity/wedge_mesh.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
+r"""
+A linear elastic beam loaded with a continuous force. The FE mesh consists
+of hexehedral and tetrahedral elements.
```
Comment (rc) [inline sfepy/examples/linear_elasticity/wedge_mesh.py:3]: Mention the wedge ones here too :)

File: sfepy/examples/linear_elasticity/wedge_mesh.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Running the simulation::
+
+    sfepy-run sfepy/examples/linear_elasticity/wedge_mesh.py
+
+Viewing the results::
+
+    sfepy-view beam_h7.vtk beam_t42.vtk beam_w14.vtk -f u:s0:wu:e:p0 u:s1:wu:e:p0 u:s2:wu:e:p0 --camera-position="1.2,-0.6,0.1,0.4,0.1,-0.1,-0.2,0.1,1"
+"""
+from __future__ import absolute_import
```
Comment (rc) [inline sfepy/examples/linear_elasticity/wedge_mesh.py:16]: This is no more needed.

**Comment by rc** [comment]
Thanks! I have only noticed a few docstring issues (see comments above), otherwise LGTM!

**Comment by vlukes** [comment]
Issues are fixed.

**Pull request merged**